examples/sine_function_model_prediction/main.ipynb

```diff
@@ -0,0 +1,186 @@
rJZGLHjh1Gh5OtnDhxgp49e1KiRAm8vb0pVaoUw4YNIzY21ujQnN7o0aMpXrw4Xl5e1KlTh82bNxsdUrYyYsQIatWqRZ48eShQoABt27bl4MGDRoeV7X366aeYTCb69+9vdChOQ5IlkSUGDRpESEiI0WFkSwcOHMBsNvPTTz+xd+9evv76a3788Ufeffddo0Nzar///jsDBgxg2LBhbNu2jSpVqhAeHs6FCxeMDi3bWLVqFa+88sr/27vbkKb6Bgzg1/RG7uaWlaBSmqUpiWlSS00jlCwlCbKwPki+UIKhS1HyBUsj/CA0UFxkIZlWxiTLRpaFUeqHjEytjJAy3/ItVqmZgjPb8yGe8fh4d7BwHdx9/eB82P8czq4zhlye/58dPHnyBHV1dZiensauXbswMTEhdjSz1dzcjAsXLsDb21vsKIsKfzqATK62thapqam4ceMGPD090dbWBh8fH7FjmbUzZ86guLgYXV1dYkdZtPz8/LBlyxacPXsWwI9nQDo5OUGpVCIzM1PkdOZJp9PBzs4ODQ0N2L59u9hxzM7Xr1+xadMmnDt3Dnl5efDx8UFhYaHYsRYF3lkik/rw4QPi4+Nx5coVSKVSseP8a4yNjWHFihVix1i09Ho9WlpaEBISYhyzsLBASEgImpqaRExm3sbGxgCA310TSUxMRHh4+KzvNc0PH6RLJmMwGBAbG4uEhAQoFAr09PSIHelfobOzE2q1GiqVSuwoi9bHjx8xMzMDe3v7WeP29vbo6OgQKZV5+/79O1JSUhAYGIgNGzaIHcfsaDQatLa2orm5WewoixLvLNEvy8zMhEQiEdw6OjqgVqsxPj6OrKwssSMvSvP9nP/XwMAAwsLCEBkZifj4eJGSE/26xMREvHr1ChqNRuwoZuf9+/dITk5GRUUF/v77b7HjLEpcs0S/TKfT4dOnT4LHuLi44MCBA7h9+zYkEolxfGZmBpaWloiKikJ5ebmpoy5q8/2craysAACDg4MICgqCv78/ysrKYGHB/4V+l16vh1QqRVVVFfbu3Wscj4mJwejoKLRarXjhzFBSUhK0Wi0aGxuxdu1aseOYnVu3biEiIgKWlpbGsZmZGUgkElhYWGBqamrWPpqLZYlMpq+vD1++fDG+HhwcRGhoKKqqquDn5wdHR0cR05mXgYEBBAcHY/Pmzbh69Sr/8C0APz8/+Pr6Qq1WA/gxTbR69WokJSVxgfcCMRgMUCqVqK6uRn19Pdzc3MSOZJbGx8fR29s7aywuLg7r169HRkYGpz3ngWuWyGRWr14967VMJgMAuLq6sigtoIGBAQQFBcHZ2RkqlQo6nc64z8HBQcRki1tqaipiYmKgUCjg6+uLwsJCTExMIC4uTuxoZiMxMRHXrl2DVquFXC7H8PAwAMDGxgZLliwROZ35kMvlcwqRtbU1bG1tWZTmiWWJaJGrq6tDZ2cnOjs755RQ3jj+fQcPHoROp0NOTg6Gh4fh4+ODe/fuzVn0Tb+vuLgYABAUFDRr/NKlS4iNjf3zgYh+gtNwRERERAK4ApSIiIhIAMsSERERkQCWJSIiIiIBLEtEREREAliWiIiIiASwLBEREREJYFkiIiIiEsCyRES0AOrr6yGRSDA6Oip2FCJaYCxLRGRWZmZmEBAQgH379s0aHxsbg5OTE7Kzs03yvgEBARgaGoKNjY1Jzk9E4uEveBOR2Xnz5g18fHxQUlKCqKgoAEB0dDRevHiB5uZmWFlZiZyQiBYT3lkiIrPj7u6O/Px8KJVKDA0NQavVQqPR4PLlyz8tShkZGXB3d4dUKoWLiwtOnjyJ6elpAD+esRcSEoLQ0FDj8/Y+f/4MR0dH5OTkAJg7Ddfb24s9e/Zg+fLlsLa2hqenJ+7evWv6iyeiBccH6RKRWVIqlaiursahQ4fQ3t6OnJwcbNy48afHy+VylJWVYeXKlWhvb0d8fDzkcjnS09MhkUhQXl4OLy8vFBUVITk5GQkJCVi1apWxLP2/xMRE6PV6NDY2wtraGq9fv4ZMJjPV5RKRCXEajojMVkdHBzw8PODl5YXW1lb89df8/z9UqVTQaDR49uyZcez69euIjo5GSkoK1Go12tra4ObmBuDHnaXg4GCMjIxg2bJl8Pb2xv79+5Gbm7vg10VEfxan4YjIbJWWlkIqlaK7uxv9/f0AgISEBMhkMuP2X5WVlQgMDISDgwNkMhlOnDiBvr6+WeeLjIxEREQE8vPzoVKpjEXpnxw7dgx5eXkIDAxEbm4uXr58aZqLJCKTY1kiIrP0+PFjFBQUoKamBr6+vjh8+DAMBgNOnz6N58+fGzcAaGpqQlRUFHbv3o2amhq0tbUhOzsber1+1jknJyfR0tICS0tLvH37VvD9jxw5gq6uLuM0oEKhgFqtNtXlEpEJsSwRkdmZnJxEbGwsjh49iuDgYFy8eBFPnz7F+fPnYWdnh3Xr1hk34EexcnZ2RnZ2NhQKBdzc3NDb2zvnvGlpabCwsEBtbS2Kiorw8OFDwRxOTk5ISEjAzZs3kZaWhpKSEpNcLxGZFssSEZmdrKwsGAwG5OfnAwDWrFkDlUqF9PR09PT0zDnezc0NfX190Gg0ePfuHYqKilBdXT3rmDt37qC0tBQVFRXYuXMnjh8/jpiYGIyMjPxjhpSUFNy/fx/d3d1obW3Fo0eP4OHhseDXSkSmxwXeRGRWGhoasGPHDtTX12Pbtm2z9oWGhuLbt2948OABJBLJrH3p6ekoLS3F1NQUwsPD4e/vj1OnTmF0dBQ6nQ5eXl5ITk5GVlYWAGB6ehpbt26Fq6srKisr5yzwViqVqK2tRX9/P5YuXYqwsDAUFBTA1tb2j30WRLQwWJaIiIiIBHAajoiIiEgAyxIRERGRAJYlIiIiIgEsS0REREQCWJaIiIiIBLAsEREREQlgWSIiIiISwLJEREREJIBliYiIiEgAyxIRERGRAJYlIiIiIgEsS0REREQC/gNEqvFZcQNnLgAAAABJRU5ErkJggg==",
```

Comment:
Let's clear the output.

Reply:
I would keep it just make it smaller so you can also browse it in GH
https://github.com/deependujha/litdata/blob/example/sine-function-model-prediction/examples/sine_function_model_prediction/main.ipynb